Herebyfile.mjs

```diff
@@ -107,9 +103,12 @@ const goBuildFlags = [
 ];
 
 /**
- * @type {<T>(fn: () => T) => (() => T)}
+ * @param {() => T} fn
+ * @returns {() => T}
+ * @template T

```

Comment:
Can you move the type parameter up-front? That way it reads a little bit more like how signatures would be declared in TS?

Reply:
For some reason I thought it had to go at the bottom, since it does for typedef for some reason. Will move.